src/renderer/gdi/state.cpp

```diff
@@ -29,8 +29,10 @@ GdiEngine::GdiEngine() :
     _fInvalidRectUsed(false),
     _lastFg(INVALID_COLOR),
     _lastBg(INVALID_COLOR),
+    _lastFontItalic(false),
     _fPaintStarted(false),
-    _hfont((HFONT)INVALID_HANDLE_VALUE)
+    _hfont(nullptr),

```

Comment:
Note that this was a deliberate change - the correct error value for a font is null rather than `INVALID_HANDLE_VALUE`. This was previously generating error logs when the font got deleted.

Reply:
Woopsie. Thanks.

Reply:
Great catch!